app/controllers/api/internal/v1/representation_orders_controller.rb

```diff
@@ -35,7 +35,9 @@ def create_params
               effective_start_date
               effective_end_date
             ],
-          )
+          ).tap do |filtered|
```

Comment:
nit: a bit more explicit for the param's name:
` ).tap do |filtered_params|`
or
` ).tap do |permitted_params|`